README.md

```diff
@@ -26,8 +27,10 @@ Vault Eagle
     source ./eagle-python/bin/activate
     python ./manage.py runserver
     # **IMPORTANT**
-    # Start by visiting http://localhost:8000/reload to download blockchain data
-    # Otherwise, the root dashboard view will crash if there is no data
+    # Start by visiting http://localhost:8000/snap to download blockchain snapshot (will take a few mins)
+    # Otherwise, the root dashboard view will crash (with an IndexError)
+    # Visit http://localhost:8000/reload to download blockchain data (data download starts from block 10884500)  
+    # Otherwise, there will be no events shown for the contracts
```

Comment:
`/reload` actually includes `/snap` and `/fetch` and was just split to reduce execution time on the GAE call.  Not sure why they would behave different?

Reply:
For people who are new to the project, entering `/reload` would start first by calling `/fetch` from a block number from late 2020 - and it takes a long time for it to load for all contracts and after it has loaded all event data, it calls `/snap`.  During this time visiting the dashboard will result in a crash.
When you call `/snap` though, the dashboard loads without an error - and it takes a couple of mins to load. 
This is why I suggested first visiting `/snap`, then `/fetch`

Reply:
My main point was that it's redundant, since `/reload` performs both of those functions.  So maybe either split it into the two endpoint calls or not.  `/reload` is a bit vestigial anyway.  Not a big deal, anyway.